### PR TITLE
disabling flakey tests

### PR DESCRIPTION
**Pull request by jmarolf**
@jaredpar @jasonmalinowski @VSadov  does this needs to be ported to 15.3 as well?

### CODE REVIEW

File: src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpErrorListNetCore.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public override void ErrorLevelWarning()
             base.ErrorLevelWarning();
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.ErrorList)]
+        [Fact(Skip = "Needs investigation"), Trait(Traits.Feature, Traits.Features.ErrorList)]
```
Comment (jasonmalinowski) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpErrorListNetCore.cs]: Can we get an actual bug?

**Comment by jasonmalinowski** [comment]
@jmarolf Target this to the dev15.3-preview1 branch and we'll flow from there.

**Comment by jmarolf** [comment]
retest perf_correctness_prtest please

**Comment by jmarolf** [comment]
@VSadov looks like perf_correctness_prtest is broken in this branch

```
MSBUILD : error MSB4166: Child node "2" exited prematurely. Shutting down. Diagnostic information may be found in files in the temporary files directory named MSBuild_*.failure.txt.
14:36:18 C:\Users\dotnet-bot\.nuget\packages\Microsoft.VSSDK.BuildTools\15.0.26124-RC3\tools\VSSDK\Microsoft.VsSDK.targets(512,5): error MSB3202: The project file "..\Core\MSBuildTask\MSBuildTask.csproj" was not found. [D:\j\workspace\perf_correctn---3557172a\src\Compilers\Extension\CompilerExtension.csproj]
```

**Pull request merged**